man/man5/monitorix.conf.5

```diff
@@ -169,6 +169,13 @@ This option will fork an independent process for each graph in order to speed up
 Default value: \fIy\fP
 .RE
 .P
+.BI enable_rrd_lock
+.RS
+This option will synchronise the rrd file access by creating the file \fI/tmp/monitorix.lock\fP and use it via flock.
```

Comment:
Just noticed. At the moment the file is suffixed `.flock` in source. This needs to be fixed in either place to match. What suffix would be preferred?